### ISSUE TITLE
Integrate translation service

### ISSUE THREAD

**Issue by yurtpage** [jenspfahl/EverydayTasks#71]
Could you please integrate the project with Weblate, Transifex or Localise.com?
That will significantly simplify translation.
The hosted Weblate have some pre-moderation, the Transifex has not so friendly interface but has a good AI translation, the Localize is similar to Transifex but from my experience slightly better.
As far as I understood only the Weblate supports also trnslation of the metadata in fastlane folder.

Similarly for the second app that would be very helpful https://github.com/jenspfahl/BDT

Your apps are target for a wide audience including non techinal people who may don't now the English.

**Comment by jenspfahl** [comment]
You are right, but since all my apps are free, i need a free translation service. Weblate looks good but i would have to host it by myself which I don't have the environment for.  More suggestions are welcome.

**Comment by jenspfahl** [comment]
Awesome, thanks, will have a look into your recommendations.
